content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -10,30 +10,37 @@ desc: "Changelog of the Agent Chat API"
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
 ## [v3.2] - Developer Preview
+
 ### Added
+
 - There's a new method **Get Agents For Transfer** ([Web](/messaging/agent-chat-api/v3.2/#get-agents-for-transfer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-agents-for-transfer)).
 - There's a new method **Get Customer** ([Web](/messaging/agent-chat-api/v3.2/#get-customer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-customer)) that returns info about a given Customer.
 - There's a new push [`chat_unfollowed`](/messaging/agent-chat-api/v3.2/rtm-reference/#chat_unfollowed) complementary to the **Unfollow Chat** method.
 
 ### Changed
+
 - The **Update Agent** method was replaced with **Set Routing Status** ([Web](/messaging/agent-chat-api/v3.2/#set-routing-status) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#set-routing-status)).
 - The `agent_updated` push was replaced with [`routing_status_set`](/messaging/agent-chat-api/v3.2/rtm-reference/#routing_status_set).
 - The method **Update Customer** ([Web](/messaging/agent-chat-api/v3.2/#update-customer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#update-customer)), which used to return the Customer data structure, now has no response payload.
 - The method **Get Chat Threads** ([Web](/messaging/agent-chat-api/v3.2/#get-chat-threads) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-chat-threads)) now requires `thread_ids`. Also, it no longer returns `threads_summary` in the response.
 - The [**Property**](/messaging/agent-chat-api/v3.2/#properties) data structure now has the same format in requests as in responses and pushes.
 - There's a new parameter `id` in the [Form field](/messaging/agent-chat-api/v3.2/#form-fields) data structure (`fields.answer.id` for the `group_chooser` field type).  It's an identifier for each option Customers can choose.
 - The **Add User to Chat** method ([Web](/messaging/agent-chat-api/v3.2/#add-user-to-chat) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#add-user-to-chat)) accepts an optional parameter `require_active_thread`.
+- There's a new parameter `away` in the [Login](/messaging/agent-chat-api/v3.2/rtm-reference/#login) request. Controls whether the connection should be considered to be in the away state by the auto-away feature. Setting it to true should help avoiding those pesky chat assignments that happen right after random reconnections.
```

Comment:
I'm not so sure about the wording here, "should help" doesn't sound very confident :D but let's here what Ola has to say

Reply:
My thoughts:

1. I wonder if Changelog is the right place to describe the role of this parameter in detail. WDYT?  What's the pattern in other docs? My intuition tells me we should only highlight its role and provide more details in the desc of the Login method.
2. Yeah, _should help_ sounds like we're not sure if it works. `(...) helps avoiding...` 
3. I would replace pesky with something more common, like troublesome.

All in all, I suggest adding this to Changelog:

- There's a new parameter `away` in the [Login](/messaging/agent-chat-api/v3.2/rtm-reference/#login) request. It was introduced to avoid problematic chat assignments that happen right after random reconnections.

We could explain the rest in the method desc.

Reply:
I'm ok with the new wording.